.github/workflows/deploy.yml

```
@@ -8,8 +8,36 @@ on:
     types: trigger-deploy
 
 jobs:
+  build-image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v1
```

There is a new release:

```suggestion
        uses: actions/checkout@v2
```

I believe we can remove the fetch-depth below, too. Then it'll only fetch the most recent commit (faster). 

See https://github.com/actions/checkout/

What do you think?

---

:+1: sure.

> I believe we can remove the fetch-depth below, too. Then it'll only fetch the most recent commit (faster).

That won't work when building a newer image with the same tag (in case you change something without doing a new release, e.g. docker deps). Because the tag history will not be fetched.